doc/dap.txt

```diff
@@ -76,6 +76,9 @@ The `Adapter` needs to contain a `type`, which can be one of:
 - `server`, to indicate that nvim-dap can connect to an already-running
   debug adapter via TCP.
 
+- `pipe`, to connect to an already running debug adapter via a unix domain
```

Comment:
Does it have to be already running? Doesn't the executable field in the adapter spec make this invalid?

Reply:
I re-phrased it a bit to (hopefully) clarify it.

Reply:
Beautiful! Thanks for this :rocket: